lego/apps/users/views/user_delete.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import permissions, status, viewsets
```

Comment:
I feel like this is small enough to be in `views/users` but :shrug:

Reply:
Just followed the convention from ChangePasswordViewSet, which is also small, but is separated out in its own view file.